demo/demo.js

```diff
@@ -68,29 +81,50 @@ export function init(formDefaults, render) {
 	return syncFormData;
 }
 
+/**
+ * Translates the form/stored data into the parameters expected by index.js
+ * @param {DemoForm} formData
+ * @returns {RevSDKDemoSettings}
+ */
 function getConfig(formData) {
 	const {
 		config,
 		tokenValue,
 		tokenType,
 		tokenIssuer,
 		sourceUrl,
-		...data
+		embedType,
+		videoId,
+		webcastId,
+		baseUrl
 	} = formData
 
-	return {
-		...data,
+	/** @type {RevSDKDemoSettings} */
+	const data = {
+		baseUrl,
 		config: {
+			// include any passed in config values (hideControls, accentColor, etc)
```

Comment:
Not sure what you mean by the comment here. I would remove it, the code seems ok without it